packages/server/ui/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { render } from "react-dom";
+
+import { App } from './app';
+
+render(<App server={getServerURL}/>, document.getElementById("root"));
+
+function getServerURL(): URL {
+  let location = new URL(document.location.href);
+  let server = location.searchParams.get('server');
```

Comment:
we should put this configurable querystring param in the README for people like me who forget

Reply:
@dagda1 
> That is a pity about `graphql-ws`, was there much involved in forking and updating `express-graphql `?

There was, and in the end, I couldn't get it to work. I think we'll need a UI for this anyway, so I figure this can be the starting point.
 
> I would be inclined to put the `ui` into its own pack now and just bite the bullet. Things take root the longer they are there.

I agree and agonized over this, and believe it or not, the reason I think we should keep it in-package for now is that we are using `npm`, not `yarn`, and we have not yet started publishing our packages to `npmjs.com` yet. Even though `npm` knows to link internal monorepo packages against each other, it still loses its mind on how to do so when there is no public version available 🤦🏻 

Development got a lot easier once we removed the dev dependency on `@simulacrum/client` since it means that you can use the CLI to add and remove deps rather than editting the package.json inline. Anyway, I agree, but didn't want tackling that yak to block this feature.



> 
> I'd also have a simple test, just to check the core functionality works.
> 

👍🏻

Reply:
no problem keeping it in the same package for now.  I just get nervous when a bundler suddenly appears.  Years of webpack have scarred me.